tests/virt_autotest/hotplugging_utils.pm

```diff
@@ -0,0 +1,109 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Summary: virtualization test utilities.
+# Maintainer: Pavel Dostal <pdostal@suse.cz>, Felix Niederwanger <felix.niederwanger@suse.de>
+
+package hotplugging_utils;
+
+use base Exporter;
+use strict;
+use warnings;
+use utils;
+use version_utils;
+use testapi;
+use File::Basename;
+use Utils::Architectures;
+
+our @EXPORT = qw(set_vcpus set_guest_memory get_guest_memory get_disk_image_name reset_guest try_attach);
+
+# Subroutine to change number of vCPUs for guest
```

Comment:
Why not proper perldoc here?

Reply:
It is a small set of utils mainly self explanatory, by looking at other utils my team is using I decided to make it based on them and not use perldoc format.